values.yaml

```
@@ -44,9 +44,10 @@ syncWorkspace:
   # to Terraform Cloud (https://app.terraform.io)
   tfeAddress: null
 
-  # Log verbosity level. One of "trace", "debug", "info", "warn", or "error".
-  # Defaults to "info".
-  logLevel: null
+  # WARNING: logLevel not supported by terraform-k8s:1.0.0 version
+  # Uncomment to configure log verbosity level. One of "debug", "info", or "error".
+  # Defaults to "debug".
+  # logLevel: null
```

I'd set this to a reasonable default, like `error`. If someone uncomments it then they've read the comment above anyway.
```suggestion
  # logLevel: error
```